src/esdt/transferOwnership-esdt.ts

```diff
@@ -0,0 +1,72 @@
+import prompts, { PromptObject } from 'prompts';
+import { exit } from 'process';
+import {
+  Transaction,
+  BytesValue,
+  AddressValue,
+  Address,
+  ContractCallPayloadBuilder,
+  ContractFunction,
+  TypedValue,
+} from '@elrondnetwork/erdjs';
+
+import { areYouSureAnswer, setup, commonTxOperations } from '../utils';
+import {
+  chain,
+  shortChainId,
+  esdtTransferOwnershipGasLimit,
+  builtInEsdtSC,
+} from '../config';
+
+const promptQuestions: PromptObject[] = [
+  {
+    type: 'text',
+    name: 'ticker',
+    message: 'Please provide the token ticker\n',
+    validate: (value) => (!value ? 'Required!' : true),
+  },
+
+  {
+    type: 'text',
+    name: 'address',
+    message: 'Please provide the address to transfer. Can be also yours\n',
```

Comment:
here the message should be something like: "Please provide the address of a new owner"